nav2_rviz_plugins/src/costmap_cost_tool.cpp

```diff
@@ -89,6 +89,7 @@ void CostmapCostTool::callCostService(float x, float y)
   auto request = std::make_shared<nav2_msgs::srv::GetCost::Request>();
   request->x = x;
   request->y = y;
+  // request->use_footprint = true;
```

Comment:
thanks, you can remove that line, it shouldn't be there

Reply:
So I actually added this line to get the footprint cost using the rviz tool. I guess there is no easy way to specify from rviz whether to get footprint or point cost so I see 3 options:

1- Get and print both
2- Leave it commented out for users to manually comment out if needed
3- Have 2 different rviz buttons

I tend towards option 1. You?

Reply:
> Get and print both

How do you specify orientation? I'm not sure that's overly useful without orientation for non-circular footprints. I think you'd need to specify an arrow like the initial pose or 2D goal tools

Reply:
ah right so we would need a separate tool for that. Ok forget it, out of scope of this PR